admin/app/components/solidus_admin/sidebar/component.html.erb

```diff
@@ -4,13 +4,13 @@
   col-start-1 col-end-2
   lg:col-start-1 lg:col-end-3
   bg-gray-15
-  p-4
+  p-8
 ">
-  <%= image_tag @logo_path, alt: "Solidus" %>
+  <%= image_tag @logo_path, alt: "Solidus", class: "py-3 px-2" %>
```

Comment:
nit: should we use the current_store name as the `alt` text instead?

Reply:
Thanks for raising that, @elia :slightly_smiling_face: Please, let me know what do you think about this solution: https://github.com/solidusio/solidus/pull/5262